invenio_previewer/templates/invenio_previewer/csv_bar.html

```diff
@@ -34,7 +34,7 @@
 {%- endblock %}
 
 {% block panel %}
-  <div id="d3_csv_previewer" data-csv-source="{{ file.uri }}"
+  <div id="d3_csv_previewer" data-csv-source="{{ url_for('invenio_records_ui.recid_download', pid_value=file.pid.pid_value, filename=file.file.key) }}"
```

Comment:
@lnielsen Here it is the same issue with the URL generation. I can look for another way to send back the document, but the problem will be there for the PNG and JPG images previewer.